org.lflang/src/org/lflang/FileConfig.java

```diff
@@ -16,6 +16,8 @@
 
 
 import org.lflang.generator.GeneratorUtils;
+import org.lflang.generator.cpp.CppFileConfig;
```

Comment:
This is not going to break the Eclipse build. Currently our java files cannot import from Kotlin without breaking the Eclipse build...

Reply:
Good catch. I've revised this in 1cc48459e.